test/assertions_test.rb

```diff
@@ -87,13 +87,43 @@ def test_assert_statsd_call
       end
     end
 
+    assert_no_assertion_triggered do
+      foo_1_metric_tags = { times: 2, tags: ['foo:1'] }
+      foo_2_metric_tags = { times: 1, tags: ['foo:2'] }
+      @test_case.assert_statsd_increments('counter', [foo_1_metric_tags, foo_2_metric_tags]) do
+        StatsD.increment('counter', tags: { foo: 1 })
+        StatsD.increment('counter', tags: { foo: 1 })
+        StatsD.increment('counter', tags: { foo: 2 })
+      end
+    end
```

Comment:
@fw42, @wvanbergen, this is the sort of thing I'd like to add support for .. does it make sense to you guys? If there's a better/easier way to do this with what we already have, happy go to that route instead.